Assets/Inventory/PlayerData.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
player data is what the user has in the inventory. we may want to save this in the user profile or just load some basic defaults. there is no persistence atm